pkg/app/sensor/app.go

```diff
@@ -27,13 +28,15 @@ import (
 	"github.com/docker-slim/docker-slim/pkg/sysenv"
 	"github.com/docker-slim/docker-slim/pkg/system"
 	"github.com/docker-slim/docker-slim/pkg/util/errutil"
+	"github.com/docker-slim/docker-slim/pkg/util/fsutil"
 	"golang.org/x/sys/unix"
 
 	log "github.com/sirupsen/logrus"
 )
 
 const (
 	defaultArtifactDirName = "/opt/dockerslim/artifacts"
+	defaultEventsFileName  = "/opt/dockerslim/events.json"
```

Comment:
is there a specific reason why it's not `/opt/dockerslim/artifacts/events.json`?